dawn/src/dawn/Serialization/SIRSerializer.cpp

```diff
@@ -466,6 +466,11 @@ static std::shared_ptr<sir::Expr> makeExpr(const dawn::proto::statements::Expr&
     return std::make_shared<sir::LiteralAccessExpr>(
         exprProto.value(), makeBuiltinTypeID(exprProto.type()), makeLocation(exprProto));
   }
+  case dawn::proto::statements::Expr::kReductionOverNeighborExpr: {
+    const auto& exprProto = expressionProto.reduction_over_neighbor_expr();
+    return std::make_shared<sir::ReductionOverNeighborExpr>(
+        exprProto.op(), makeExpr(exprProto.rhs()), makeExpr(exprProto.init()));
+  }
```

Comment:
did I miss the serialization part? seems like only deserialization is here. - Which leads me to the question: Do we want to have a tests-case for this also in Serializer and maybe even one in the (dawn)-integration tests? I'd suggest to at least move the latter to a separate task, but maybe the former does makes sense?

Reply:
done